src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1990,9 +2096,13 @@ static fsal_status_t cfs_file_open(struct kvsfs_file_state *state,
 static fsal_status_t cfs_file_close(struct kvsfs_file_state *state,
 				    struct kvsfs_fsal_obj_handle *obj)
 {
+	fsal_status_t result;
+	perfc_trace_inii(PFT_CFS_CLOSE, PEM_CFS_TO_NFS);
```

Comment:
CFS tags should not be used from this file

Reply:
which is convenient function tag to use from handle.c?

Reply:
Same as previous